cmd/cbft/main.go

```diff
@@ -888,6 +888,26 @@ func (meh *mainHandlers) OnRefreshManagerOptions(options map[string]string) {
 			log.Printf("main: meh.OnRefreshManagerOptions, err: %v", err)
 			return
 		}
+		if ftsMemQuota, exists := options["ftsMemQuota"]; exists {
```

Comment:
Consider if this flow on refreshing options should be removed?
Each time FTS mem quota is changed from the UI, it leads to a restart, followed by initMemOptions(), where this condition is checked.